### ISSUE TITLE
How to build the build-chain as a cross-compiler? The link "to build" is broken

### ISSUE THREAD

**Issue by Fabrizio-Caruso** [tkchia/gcc-ia16#153]
Hi! I am trying to figure out what I need to build gcc-ia16 on my PC. My goal is to use it as a cross-compiler that I can run through the command line to target 16-bit DOS.

I do not find the instructions. I see several versions of this repos in different places. Which one is the most recent and the one that is still maintained?
The link shown on the main page here and which is supposed to point to build instructions is broken: https://github.com/tkchia/build-ia16

![image](https://github.com/user-attachments/assets/8452c9ef-257a-4822-ba38-955ef6d372c8)

**Comment by lpsantil** [comment]
The projects moved to gitlab a while back - https://gitlab.com/tkchia/build-ia16

**Comment by Fabrizio-Caruso** [comment]
@lpsantil I know. It has even a third mirror. They are all synchronized. 
My question is how do I build it? The gitlab page has a diagram, with several steps about things I don't understand.
I just want to build as a cross-compiler on PC (under Linux or better Cygwin POSIX).
Could you please tell me which commands I have to run?

**Comment by lpsantil** [comment]
Probably better to ask on Gitlab but I assume follow the flow of the build.sh script 



```
./fetch.sh
./build.sh clean
./build.sh binutils 
./build.sh prereqs 
./build.sh gcc1
./build.sh newlib
./build.sh causeway
./build.sh libi86
./build.sh clean-djgpp
./build.sh prereqs-djgpp
./build.sh binutils-djgpp
./build.sh gcc-djgpp
./redist-djgpp.sh all
```

**Comment by asiekierka** [comment]
Yes, that's more or less correct; you follow the diagram step by step from top to bottom; arrows which go to the side are optional components.
